nengo_gui/examples/tutorial/20-spa-actions.py

```diff
@@ -0,0 +1,61 @@
+# Tutorial 20: Semantic Pointer Actions
+
+# A complex cognitive system needs a method to select and perform actions.
+# In particular, it is useful to have a model that can do one thing at
+# a time, sequentially.  There is significant psychological data indicating
+# that people have such a system, providing a serial bottleneck of
+# cognition.
+#
+# In mammals, this system is often identified with the Basal Ganglia.
+# We have shown that a Nengo model of the Basal Ganglia acting as
+# an action selection network matches well to human data:
+# http://compneuro.uwaterloo.ca/files/publications/stewart.2012a.pdf
+#
+# This model has been built in to the Nengo SPA system for use in your
+# models.  To use it, for every action you need to define some function
+# that computes the utility of the action (how good the action is right
+# now) and what the effect of the action should be (what to do when
+# the action is chosen).  The Basal Ganglia model will monitor the
+# utilities of each action and select the one with the highest value.
+#
```

Comment:
Would it be useful to add a line about how thalamus is used to implement the effects for the bg?